cocos/renderer/CCTexture2D.cpp

```diff
@@ -1444,4 +1464,9 @@ void Texture2D::removeSpriteFrameCapInset(SpriteFrame* spriteFrame)
     }
 }
 
+void Texture2D::setAutoGenerateMipmap(bool generateMipmaps)
+{
+    _generateMipmaps = generateMipmaps;
```

Comment:
Yes.
Old version need call generateMipmap after initWithMipmaps, if want to generate mipmap explicitly.
New version need call setAutoGenerateMipmap before initWithMipmaps by developers if want to generate mipmap explicitly.